src/darwin/Framework/CHIP/zap-generated/MTRBaseClusters.h

```diff
@@ -5799,21 +6229,42 @@ light or a window shade.
 
 /**
  * Cluster Group Key Management
- *    The Group Key Management Cluster is the mechanism by which group keys are managed.
+ *
+ * The Group Key Management Cluster is the mechanism by which group keys are managed.
  */
 @interface MTRBaseClusterGroupKeyManagement : MTRCluster
 
 - (instancetype _Nullable)initWithDevice:(MTRBaseDevice *)device
                               endpointID:(NSNumber *)endpointID
                                    queue:(dispatch_queue_t)queue NS_DESIGNATED_INITIALIZER MTR_NEWLY_AVAILABLE;
 
+/**
+ * Command KeySetWrite
+ *
+ * Revoke a Root Key from a Group
+ */
 - (void)keySetWriteWithParams:(MTRGroupKeyManagementClusterKeySetWriteParams *)params
                    completion:(MTRStatusCompletion)completion MTR_NEWLY_AVAILABLE;
+/**
+ * Command KeySetRead
+ *
+ * Revoke a Root Key from a Group
```

Comment:
Copy and paste? SetWrite/Read/Remove all claim to revoke a Root key. But read is for sure not doing that.

Reply:
```
% git grep "Revoke a Root Key from a Group"
src/app/zap-templates/zcl/data-model/chip/group-key-mgmt-cluster.xml:      <description>Revoke a Root Key from a Group</description>
src/app/zap-templates/zcl/data-model/chip/group-key-mgmt-cluster.xml:      <description>Revoke a Root Key from a Group</description>
src/app/zap-templates/zcl/data-model/chip/group-key-mgmt-cluster.xml:      <description>Revoke a Root Key from a Group</description>
```

Filed https://github.com/project-chip/connectedhomeip/issues/23892